transports/quartz/src/main/java/org/mule/transport/quartz/QuartzMessageDispatcher.java

```diff
@@ -58,7 +58,7 @@ protected void doDispatch(MuleEvent event) throws Exception
             throw new IllegalArgumentException(CoreMessages.objectIsNull(QuartzConnector.PROPERTY_JOB_CONFIG).getMessage());
         }
 
-        JobDetail jobDetail = new JobDetail();
+        JobDetailImpl jobDetail = new JobDetailImpl();
```

Comment:
I understand that now you have to implement the Impl class... but if possible, keep the jobDetail variable with the highest level type as possible